python/servo/testing_commands.py

```diff
@@ -486,42 +489,55 @@ def create_parser_create():
 @CommandProvider
 class WebPlatformTestsCreator(CommandBase):
     template_prefix = """<!doctype html>
-%(documentElement)s<meta charset=utf-8>
+%(documentElement)s<meta charset="utf-8">
 """
     template_long_timeout = "<meta name=timeout content=long>\n"
 
     template_body_th = """<title></title>
-<script src=/resources/testharness.js></script>
-<script src=/resources/testharnessreport.js></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
 <script>
 
 </script>
 """
 
     template_body_reftest = """<title></title>
-<link rel=%(match)s href=%(ref)s>
+<link rel="%(match)s" href="%(ref)s">
 """
 
     template_body_reftest_wait = """<script src="/common/reftest-wait.js"></script>
 """
 
-    def rel_path(self, path):
-        if path is None:
-            return
+    def make_test_file_url(self, absolute_file_path):
+        # Make the path relative to the project top-level directory so that
+        # we can more easily find the right test directory.
+        file_path = os.path.relpath(absolute_file_path, PROJECT_TOPLEVEL_PATH)
 
-        abs_path = os.path.normpath(os.path.abspath(path))
-        return os.path.relpath(abs_path, os.path.abspath(os.path.join(here, "..", "..")))
+        if file_path.startswith(WEB_PLATFORM_TESTS_PATH):
```

Comment:
Oh you're right. I forgot `lstrip` param takes a set of characters instead of just a string to strip